XamarinCommunityToolkitSample/Pages/Behaviors/ImpliedOrderGridBehaviorPage.xaml

```diff
@@ -0,0 +1,53 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<pages:BasePage xmlns="http://xamarin.com/schemas/2014/forms"
+          xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+          xmlns:pages="clr-namespace:Xamarin.CommunityToolkit.Sample.Pages;assembly=Xamarin.CommunityToolkit.Sample"
+          xmlns:behaviors="clr-namespace:Xamarin.CommunityToolkit.Behaviors;assembly=Xamarin.CommunityToolkit"
+          xmlns:behaviors1="clr-namespace:Xamarin.CommunityToolkit.Sample.Pages.Behaviors;assembly=Xamarin.CommunityToolkit.Sample"
+          x:Class="Xamarin.CommunityToolkit.Sample.Pages.Behaviors.ImpliedOrderGridBehaviorPage">
+    <StackLayout>
+        <StackLayout Orientation="Horizontal" Spacing="5">
+            <Button Text="Add Row" Clicked="ButtonAddRow_OnClicked"></Button>
+            <Button Text="Add Column" Clicked="ButtonAddColumn_OnClicked"></Button>
+            <Button Text="Remove Row" Clicked="ButtonRemoveRow_OnClicked"></Button>
+            <Button Text="Remove Column" Clicked="ButtonRemoveColumn_OnClicked"></Button>
+        </StackLayout>
+
+        <Label Text="View's row and column are determined by the order they are added to the grid's children.  RowSpan and ColumnSpan are honored."></Label>
```

Comment:
One extra space "...children.  RowSpan..." here
Could you please move all strings to Resx file? 👍

Reply:
Extra space is a long-standing habit from being taught to type on an electric typewriter :)